src/Table/HeaderCell.jsx

```diff
@@ -25,14 +26,19 @@ export default function HeaderCell({
         sortable && cssClass.SORTABLE,
         className
       )}
-      noWrap
       onClick={() => sortable && onSortChange()}
       width={width}
     >
-      <div className={cssClass.LABEL}>
-        {children}
-      </div>
-      {sortable && <SortIcons direction={activeSortDirection} className={cssClass.SORT} />}
+      <FlexBox alignItems={ItemAlign.CENTER} >
```

Comment:
fwiw, table cells already have built-in vertical alignment, so in theory, you should be able to do `vertical-align: middle` at the cell level to avoid having to use a flexbox